packages/nimble-components/src/chip/styles.ts

```diff
@@ -0,0 +1,98 @@
+import { css } from '@ni/fast-element';
+import {
+    actionRgbPartialColor,
+    bodyDisabledFontColor,
+    bodyFont,
+    bodyFontColor,
+    borderRgbPartialColor,
+    borderWidth,
+    controlHeight,
+    iconColor,
+    iconSize,
+    mediumPadding,
+    smallPadding
+} from '../theme-provider/design-tokens';
+import { display } from '../utilities/style/display';
+import { appearanceBehavior } from '../utilities/style/appearance';
+import { ChipAppearance } from './types';
+
+export const styles = css`
+    ${display('inline-flex')}
+
+    :host {
+        height: ${controlHeight};
+        width: fit-content;
+        max-width: 300px;
+        color: ${bodyFontColor};
+        font: ${bodyFont};
+        padding: 0 ${mediumPadding};
+        gap: 4px;
+        background-color: transparent;
+        border: ${borderWidth} solid rgba(${actionRgbPartialColor}, 0.3);
+        border-radius: 4px;
+        justify-content: center;
+        align-items: center;
+        background-repeat: no-repeat;
+    }
+
+    :host([disabled]) {
+        cursor: default;
+        color: ${bodyDisabledFontColor};
+        border-color: rgba(${borderRgbPartialColor}, 0.3);
+        box-shadow: none;
+        background-image: none;
+        background-size: 100% 100%;
+    }
+
+    :host([disabled])::before {
+        box-shadow: none;
+    }
+
+    :host([disabled]) slot[name='start']::slotted(*),
+    :host([disabled]) .remove-button {
+        opacity: 0.3;
+        ${iconColor.cssCustomProperty}: ${bodyFontColor};
+    }
+
+    slot[name='start']::slotted(*) {
+        flex-shrink: 0;
+    }
+
+    [part='start'] {
+        display: contents;
+        ${iconColor.cssCustomProperty}: ${bodyFontColor};
+    }
+
+    .content {
+        text-overflow: ellipsis;
+        overflow: hidden;
+        white-space: nowrap;
+    }
+
+    .remove-button {
+        height: ${iconSize};
+        width: ${iconSize};
+        margin-right: calc(-1 * ${smallPadding});
+    }
+`.withBehaviors(
+    appearanceBehavior(
+        ChipAppearance.block,
+        css`
+            :host {
+                background-image: linear-gradient(
```

Comment:
Why did you need to set a background-image to achieve a solid background color?

Reply:
This is what the button does. From what I can this is done to remove the border?

With `background-image`:
<img width="287" height="92" alt="image" src="https://github.com/user-attachments/assets/baa1e1d6-7f7a-427f-9ba9-f4c632d587e4" />

Using `background-color = `:
<img width="287" height="92" alt="image" src="https://github.com/user-attachments/assets/a76b718b-a34a-41e9-9149-bed93fd87cbf" />

Design:
<img width="224" height="87" alt="image" src="https://github.com/user-attachments/assets/74b25e30-4c0c-4769-8f00-24a56581e1fd" />

I'll try a different route, however, and just have a transparent border, so we don't see that fuzziness at the corners.

Reply:
Fixed.